.phan/config.php

```diff
@@ -23,7 +23,8 @@
 		"PhanTypeMismatchArgument",
 		"PhanTypeMismatchReturn",
 		"PhanTypeMismatchProperty",
-        "PhanTypeSuspiciousStringExpression"
+        "PhanTypeSuspiciousStringExpression",
+        "PhanAccessMethodInternal",
```

Comment:
Why is this added?

Reply:
fixing the phan error inside the PHPUnit 8 core library.